src/ui/view/vault.rs

```diff
@@ -183,19 +184,19 @@ impl VaultOnChainTransactionsPanel {
                         Row::new()
                             .push(
                                 Container::new(text::simple(
-                                    "This vault needs to be acknowledged ",
+                                    "Do you want to create of vault from this deposit ? ",
```

Comment:
Should be "Do you want to create **a** vault", and also please remove the whitespace before and after the "?"